### PR TITLE
NEPT-1134: Textarea component.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: templates/textarea/textarea.tpl.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * @file
+ * Contains template file.
+ */
+?>
+<textarea<?php print $attributes; ?>><?php print check_plain(render($element['#value'])); ?></textarea>
```
Comment (Fefaine) [inline templates/textarea/textarea.tpl.php]: I don't like very much having the check_plain in the tpl, but i don't have ideas how to do otherwise

Comment (drupol) [reply templates/textarea/textarea.tpl.php]: You're perfectly right, I must have forgot this part. I will provide an update.

Comment (Medi4tor) [reply templates/textarea/textarea.tpl.php]: Sorry, I missed that also..

Comment (drupol) [reply templates/textarea/textarea.tpl.php]: No worries, I fixed it :-)

**Pull request merged**